mobile-app/app/screens/AppNavigator/screens/Auctions/components/EmptyBidsScreen.tsx

```diff
@@ -0,0 +1,40 @@
+import { tailwind } from '@tailwind'
+import { ThemedIcon, ThemedText, ThemedView } from '@components/themed'
+import { translate } from '@translations'
+import * as React from 'react'
+import { InfoTextLink } from '@components/InfoTextLink'
+import { View } from 'react-native'
+
+export function EmptyBidsScreen (): JSX.Element {
+  return (
+    <ThemedView
+      style={tailwind('px-8 mt-8 pb-2 pt-32 text-center')}
+      testID='empty_bid'
+    >
+      <ThemedIcon
+        light={tailwind('text-black')}
+        dark={tailwind('text-white')}
+        iconType='MaterialIcons'
+        name='no-sim'
+        size={32}
```

Comment:
```suggestion
        size={44}
```
To match empty vault UI